pkg/sdn/plugin/controller.go

```diff
@@ -457,8 +461,17 @@ func (plugin *OsdnNode) AddServiceRules(service *kapi.Service, netID uint32) {
 	glog.V(5).Infof("AddServiceRules for %v", service)
 
 	otx := plugin.ovs.NewTransaction()
+	action := getServiceRuleAction(netID)
+
+	// Add blanket rule allowing subsequent IP fragments
+	if netID != osapi.GlobalVNID {
```

Comment:
VNID 0 is only special with the multitenant plugin (ie, not the networkpolicy plugin), so drop this check. (The rule it adds will end up being unused in multitenant, but that's consistent with other VNID 0 rules.)